content/en/getting_started/agent/_index.md

```diff
@@ -13,42 +13,71 @@ further_reading:
       text: 'Why should I install the Datadog Agent on my cloud instances?'
 ---
 
-This guide provides an introduction to the Agent and how you can use it to send system-level metrics to the Datadog platform. It walks through an example Agent installation on Ubuntu. It covers:
+## Overview
 
-  - Agent installation
-  - Verifying that the Agent is running
-  - Configuring Agent features
-  - Troubleshooting resources
+This guide introduces the Datadog Agent and covers:
 
-## Overview
+  - [Introduction to the Agent](#the-datadog-agent)
+  - [Installation](#installing-the-agent)
+  - [Data collected by the Agent](#data-collected-by-the-agent)
+  - [Advanced configurations and features](#advanced-configurations-and-features)
+  - [Troubleshooting](#troubleshooting)
 
-### About the Agent
 
-The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. It can run on your local hosts (Windows, MacOS), containerized environments (Docker, Kubernetes), and in on-premises data centers. You can install and configure it using configuration management tools (Chef, Puppet, Ansible).
+## What is the Datadog Agent?
 
-The Agent is able to collect 75 to 100 system-level metrics every 15 to 20 seconds. With additional configuration, the Agent can send live data, logs, and traces from running processes to the Datadog Platform. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][1].
+The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. 
 
-### Agent overhead
+The Agent can run on:
+- Local hosts (Windows, MacOS), 
+- Containerized environments (Docker, Kubernetes),
+- On-premises data centers. 
```

Comment:
Avoid separating bullet lists with [commas](https://datadoghq.atlassian.net/wiki/spaces/WRITING/pages/5341511907/Lists?atlOrigin=eyJpIjoiMGU5MzhjYmQ3MGUxNGViOTkxYjliNzBkNGM2ZTA3Y2EiLCJwIjoiYyJ9#:~:text=Do%20not%20use%20semicolons%2C%20commas%2C%20or%20conjunctions%20to%20separate%20the%20bulleted%20or%20numbered%20list%20items.). Lists that read like one continuous sentence can be harder to scan.

Also note the capitalization of macOS:
```suggestion
The Agent can run on:
- Local hosts (Windows, macOS)
- Containerized environments (Docker, Kubernetes)
- On-premises data centers
```